web-marketplace/src/pages/ChooseCreditClass/ChooseCreditClass.tsx

```diff
@@ -74,7 +74,7 @@ const ChooseCreditClass: React.FC<React.PropsWithChildren<unknown>> = () => {
   return (
     <ProjectFormAccessTemplate
       loading={isFetching}
-      project={project}
```

Comment:
should we use suggestions for dev/build from https://vitejs.dev/guide/features.html#transpile-only to avoid this type of issue in the future? this is not the first time that I'm missing such an issue.

Reply:
Adding `tsc --noEmit` in the build script command should be enough.